.gitignore

```diff
@@ -21,6 +20,7 @@
 *.utf8.md
 *.knit.md
 inst/doc
-
 .DS_Store
 .Rproj.user
+.Rprofile
+renv
```

Comment:
IDK what renv is, but .Rprofile could go into the .Rbuildignore of the template repository.

Reply:
renv is the new packrat